qemu/tests/check_reports_end_offset.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from virttest import data_dir
+from virttest.qemu_storage import QemuImg
+
+
+def run(test, params, env):
+    """
+    'qemu-img check' should report the end offset of the image.
+    1. Create a test qcow2 image.
+    2. Check there is image end offset and the value with both "humam"
+        and "json" output.
+    :param test: Qemu test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    def _check_output(image, outputs, key_strings):
+        """Check the info of image end offset."""
+        for output, key_string in zip(outputs, key_strings):
+            logging.info(
+                "Check the info of image end offset with %s output." % output)
+            check_result = image.check_image(image.params, root_dir,
+                                             output="%s" % output).stdout.decode()
+            if key_string not in check_result:
+                test.fail("The check output is not correct. It's : %s."
+                          % check_result)
+
+    foo = params["images"]
+    root_dir = data_dir.get_data_dir()
+    foo = QemuImg(params.object_params(foo), root_dir, foo)
+
+    logging.info("Create the test image file.")
+    foo.create(foo.params)
+
+    _check_output(foo, ["human", "json"],
```

Comment:
Please don't hardcode the offset in your case, based on your testcase, you should check two things here:
1. there is `image end offset` field in the output.
2. the `image end offset` value remains the same.
Your hardcoded offset may result in test failures if future `qcow2` image actual size changes.